### PR TITLE
Update support table and what's new for 0.26.0

### PR DESCRIPTION
**Pull request by SueChaplain**
- release topic: point to 0.25.0 for OpenJDK 8 & 11 builds, for features now fully tested
- support topic: remove "future" note for 0.26.0 and remove OpenJDK 15 column from table

Signed-off-by: SueChaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
@pshipton - are there any OSs going EOL that we need to be aware of in the support tables. I noted Ubuntu 16.04 was going out, but we're already on 18.04 minimum. I see some discussions about centos6/rhel6?

**Comment by pshipton** [comment]
RHEL/CentOS 7.6 ends May 31, 2021, we can bump that up to 7.7
I don't think support for 6 can be removed atm.

File: docs/openj9_support.md [hunk 1]
```diff
@@ -43,16 +43,16 @@ columns will be removed over time.
...
+| v 0.26.0       | Apr 2021          | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |                                                                                    |
+| v 0.27.0       | Jul 2021 (1)      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |                                                                                    |
+| v 0.28.0       | Sep 2021 (1)      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no </span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> (3) |
```
[the remark this answers is not in the document]
Comment (pshipton) [reply docs/openj9_support.md:55]: Oops, I had this comment pending although I thought I'd submitted it.

[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/openj9_support.md]: I had missed that the 0.28 row was wrong. Fixed and added the October 2021 row. Removed the first 2 rows before Oct 2020 to keep the table a reasonable size.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Review by pshipton — APPROVED** [review]

**Pull request merged**